src/Core/Services/IPaymentService.cs

```diff
@@ -15,6 +15,7 @@ public interface IPaymentService
            short additionalStorageGb, int additionalSeats, bool premiumAccessAddon, TaxInfo taxInfo);
         Task<string> PurchasePremiumAsync(User user, PaymentMethodType paymentMethodType, string paymentToken,
             short additionalStorageGb, TaxInfo taxInfo);
+        Task<string> AdjustSeatsAsync(Organization organization, Models.StaticStore.Plan plan, int additionalSeats);
```

Comment:
Lets import `StaticStore` instead of referencing it via Models inline here and in `UpgradeFreeOrganizationAsync`